code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -376,6 +376,9 @@
 
 /datum/action/innate/ai/destroy_rcds/Activate()
 	for(var/obj/item/rcd/RCD in GLOB.rcd_list)
+		var/turf/RCD_turf = get_turf(RCD)
+		if(RCD_turf.z != usr.z)
```

Comment:
might be better to check for station Z level. just my opinion

Reply:
technicality resolved with latest commit.